kentikapi/client_integration_test.go

```diff
@@ -265,7 +265,7 @@ func TestClient_GetAgentWithRetries(t *testing.T) {
 			expectedError:     true,
 		}, {
 			name:              "timeout during first request",
-			timeout:           pointer.ToDuration(1 * time.Millisecond),
+			timeout:           pointer.ToDuration(5 * time.Millisecond),
```

Comment:
This may still timeout depending on system state at the moment of test execution, but it is so lightweight fix that lets give it a try